customer-terminate-service/domain/src/test/java/se/inera/intyg/cts/domain/repository/InMemoryCertificateBatchRepository.java

```diff
@@ -10,6 +11,12 @@ public class InMemoryCertificateBatchRepository implements CertificateBatchRepos
   private CertificateBatch certificateBatch;
   private List<CertificateText> certificateTexts;
 
+  @Override
+  public CertificateSummary certificateSummary(
+      Termination termination) {
```

Comment:
Is termination used in this method?

Reply:
No, it is just the dummy implementation in the test package.